generator/codegen/collection_extensions.go

```diff
@@ -313,7 +315,13 @@ func GetCollections(swagger *openapi3.T) (collections CollectionParams, err erro
 						continue
 					}
 
-					collectionSet.Add(newCollection(collectionRef, itemRef))
+					isNoPaginationCollection, subErr := isExtensionFlagSet(schemaVal.ExtensionProps, noPaginationFlag)
+					if subErr != nil {
+						err = subErr
+						return
+					}
+
+					collectionSet.Add(newCollection(collectionRef, itemRef, isNoPaginationCollection))
```

Comment:
Whilst more verbsoe, I think the code will be clearer if you pull out the `isNoPaginationCollection` into an explicit check